### PR TITLE
separate in diffrent components

### PR DESCRIPTION
**Pull request by nikuhb**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <div> 
-  <div class="bg-warning"> horizontal bar </div>
+  <div class="bg-warning">
+    <app-main-nav-bar></app-main-nav-bar>
+  </div>
   <div class="d-flex">
-    <div class="bg-primary"> left panel </div>
+    <div class="bg-primary">
```
Comment (andrespipe) [inline src/app/app.component.html:6]: Remove the bg-primary class you don't need that

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,7 +1,11 @@
 <div> 
-  <div class="bg-warning"> horizontal bar </div>
+  <div class="bg-warning">
```
Comment (andrespipe) [inline src/app/app.component.html:2]: Remove the bg-warning class you don't need that

File: src/app/components/main-nav-bar/main-nav-bar.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<p>main-nav-bar works!</p>
```
Comment (andrespipe) [inline src/app/components/main-nav-bar/main-nav-bar.component.html:1]: Please start to create the nav bar, this element is completely empty

File: src/app/components/left-panel/left-panel.component.html [hunk 4]
```diff
@@ -0,0 +1 @@
+<p>left-panel works!</p>
```
Comment (andrespipe) [inline src/app/components/left-panel/left-panel.component.html:1]: Please after you create the nav bar start to work on this component is completely empty

File: src/app/app.component.html [hunk 5]
```diff
@@ -1,7 +1,11 @@
...
+    <app-main-nav-bar></app-main-nav-bar>
+  </div>
   <div class="d-flex">
-    <div class="bg-primary"> left panel </div>
+    <div class="bg-primary">
+      <app-left-panel></app-left-panel>
+    </div>
     <div class="bg-secondary flex-grow-1"> main panel </div>
```
Comment (nikuhb) [inline src/app/app.component.html:9]: Remove classes that you don't need:

bg-warning
bg-primary
bg-secondary

Comment (andrespipe) [inline src/app/app.component.html]: Remove classes that you don't need:

bg-warning
bg-primary
bg-secondary

File: src/app/components/left-panel/left-panel.component.html (hunk 4 above)
Comment (andrespipe) [inline src/app/components/left-panel/left-panel.component.html]: Please after you create the nav bar start to work on this component is completely empty

File: src/app/components/main-nav-bar/main-nav-bar.component.html (hunk 3 above)
Comment (andrespipe) [inline src/app/components/main-nav-bar/main-nav-bar.component.html]: Please start to create the nav bar, this element is completely empty

**Pull request merged**